db/migrate/20170817035830_provision_stripe_payment_settings_to_all_users.rb

```diff
@@ -0,0 +1,33 @@
+class ProvisionStripePaymentSettingsToAllUsers < ActiveRecord::Migration[5.1]
+  def up
+    communities_with_stripe_ids    = Community.connection.select_values("SELECT community_id FROM payment_settings WHERE payment_gateway = 'stripe'")
```

Comment:
Models shouldn't be used in migrations. They are evolving and might break the migrations later. Community might be a quite safe bet, but it's a good general rule to either not use them or circumvent the issue by defining the model explicitly in the migration. So adding lines:

```ruby
class Community < ActiveRecord::Base
end
```
would fix the issue.